docs/exploring/Simplify-GenAI-RAG-Deployments-with-Open-Platform-for-Enterprise-OPEA.markdown

```diff
@@ -0,0 +1,44 @@
+
+## ChatQnA Reference Architecture
+
+Among the multiple [examples OPEA provides](https://github.com/opea-project/GenAIExamples) to help with deployments of GenAI applications ([CodeGen](https://github.com/opea-project/GenAIExamples/tree/main/CodeGen) or [VisualQnA](https://github.com/opea-project/GenAIExamples/tree/main/VisualQnA)), we will explore the RAG chatbot ([ChatQnA](https://github.com/opea-project/GenAIExamples/tree/main/ChatQnA)) as a prototypical example, composed of several independent modules, each tightly coordinated to handle specific functions or tasks, as shown in the image:
+
+![ChatQnA OPEA Blueprint]({{site.baseurl}}/assets/images/opea_arq.png)
+The reference architecture is divided into 3 parts:
+
+* **Conversational UI**: The chatbot interface includes an input field for user queries, which displays responses and passes the user inputs to the backend (Megaservice) or RAG context. 
+
+* **ChatQnA Megaservice**: This group of microservices receives the user message from the UI and starts the RAG process. The megaservice is a higher-level construct that assembles multiple microservices (retriever, embeddings, reranking, LLM) to deliver end-to-end applications. Unlike microservices, which handle specific tasks, this megaservice will orchestrate the microservices’ interactions to provide a complete solution, encapsulating complex business logic and workflows. This modular approach integrates microservices as a unit to fulfill specific application requirements.
+
+* **RAG Context**: The example allows the user to upload a file to be used as context for future queries. The dataprep microservice performs the preprocessing and inserts it into the vector database to be used to provide context for future queries.
+Each microservice is available as a Docker container. This allows any GenAI example to be deployed using Docker Compose or a Kubernetes cluster.
+
+##Call to Action
+Try building your chatbot using the ChatQnA demo following the steps from the repository https://github.com/opea-project/GenAIExamples/tree/main/ChatQnA: 
+
+1. Start docker service using docker image
+ * Pull pre-built docker images from docker hub
+ * Build microservices from source
+
+2. Set environment variables
+3. Deploy end-to-end example (docker compose or Kubernetes using Helm Chart)
+4. Consume individual microservices / megaservice
+5. Monitoring with Prometheus and Grafana dashboard
+
+##Conclusion
```

Comment:
Please put a space after the `##`. Otherwise, this is not interpreted as a section title.